.cicd/build.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+set -eo pipefail
+. ./.cicd/helpers/general.sh
+
+fold-execute mkdir -p $BUILD_DIR
+
+if [[ $(uname) == 'Darwin' ]]; then
+
+    cd $BUILD_DIR
+    [[ $TRAVIS == true ]] && fold-execute ccache -s
+    fold-execute cmake -DCMAKE_BUILD_TYPE=Release ..
+    fold-execute make -j$JOBS
+
+else # Linux
+
+    MOUNTED_DIR='/workdir'
+    ARGS=${ARGS:-"--rm -v $(pwd):$MOUNTED_DIR"}
```

Comment:
We can hard-code the `$MOUNTED_DIR` path, and we can eliminate the `$ARGS` variable altogether. We do not expect either of these things to change, so they do not need to be variables.